pipelines/batch/src/main/java/com/google/fhir/analytics/ParquetMerger.java

```diff
@@ -239,8 +245,20 @@ public static void main(String[] args) throws IOException {
       throw new IllegalArgumentException("All of --dwh1, --dwh2, and --mergedDwh should be set!");
     }
 
-    Pipeline pipeline = createMergerPipeline(options, fhirContext);
-    EtlUtils.runMergerPipelineWithTimestamp(pipeline, options);
+    List<Pipeline> pipelines = createMergerPipelines(options, fhirContext);
```

Comment:
nit: Please consider refactoring this logic into a function in `EtlUtil` as well; I think this is very similar to what you do in `FhirEtl.java` above. In the javadoc of that function please also describe briefly why we need to use a low number of pipelines running in parallel.

Reply:
Done